### PR TITLE
Timeout util

### PR DESCRIPTION
**Pull request by neo-Aditya**

### CODE REVIEW

File: src/main/java/frc/robot/utils/TimeoutCounter.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+public class TimeoutCounter {
+    private int timeoutCounter = 0;
+    private final String commandName;
+    public TimeoutCounter(String commandName) {
+        this.commandName = commandName;
+        Logger.logInteger("/Timeouts/" + commandName, timeoutCounter, Constants.ENABLE_LOGGING);
+        System.out.println("Constructed");
+    }
+    public double getTimeoutCount() {
+        System.out.println("Got TimeoutCounter");
```
Comment (armadilloz) [inline src/main/java/frc/robot/utils/TimeoutCounter.java]: Please remove System.out statements.

**Review by armadilloz — CHANGES_REQUESTED** [review]
Looks goos, just one comment.

File: src/main/java/frc/robot/subsystems/TempLogging.java
Comment (bengold22) [inline src/main/java/frc/robot/subsystems/TempLogging.java:1]: Do you aim to make this a subsystem and log in periodic?

Comment (bengold22) [inline src/main/java/frc/robot/subsystems/TempLogging.java:1]: Probably shouldn't be named "Temp"

**Review by bengold22 — CHANGES_REQUESTED** [review]

**Review by bengold22 — APPROVED** [review]

File: src/main/java/frc/robot/utils/TimeoutCounter.java (hunk 1 above)
Comment (neo-Aditya) [reply src/main/java/frc/robot/utils/TimeoutCounter.java]: Done

**Review by armadilloz — APPROVED** [review]

**Pull request merged**